virt/virtual_machines/advanced_vm_management/virt-vm-control-plane-tuning.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="virt-vm-control-plane-tuning"]
+= Virtual machine control plane tuning
+
+include::_attributes/common-attributes.adoc[]
+:context: virt-control-plane-tuning
+
+OpenShift Virtualization offers several tuning options at the control-plane level. This includes:
+
+* The `highBurst` profile, which uses fixed `QPS` and `burst` rates, to create hundreds of virtual machines (VMs) in one batch.
```

Comment:
These list items aren't sentences so shouldn't have a period at the end (per IBM Style)
```suggestion
* The `highBurst` profile, which uses fixed `QPS` and `burst` rates to create hundreds of virtual machines (VMs) in one batch
```